hulks/check_django_migrations_filename.py

```diff
@@ -0,0 +1,25 @@
+import re
+import sys
+
+from hulks.base import BaseHook
+
+
+class DjangoMigrationFilenameHook(BaseHook):
+    MIGRATIONS_FILES_PATTERN = r'.*[0-9]{4}_\w+_[0-9]{8}_[0-9]{4}.py$'
```

Comment:
this pattern is very restrictive, why I should end in numbers ?
Maybe something like `r'\d{4}_\w+.py' would be enough...

Reply:
It is restrictive because we want catch default pattern django migrations:
example 0021_auto_20180713_1909


I do not know if i was clear

Reply:
Actually this is used to invalidate the migration name, right ?
Maybe it's easier to map in simple rules instead of using regex:
* startswith 4 digits
* "auto" not in migration name
* numbers not allowed in migration name (after the first `_` for ex)